assert.js

```diff
@@ -483,6 +483,18 @@ assert.doesNotThrow = function(block, /*optional*/error, /*optional*/message) {
 
 assert.ifError = function(err) { if (err) throw err; };
 
+// Expose a strict only variant of assert
+function strict(value, message) {
+  if (!value) fail(value, true, message, '==', strict);
+}
+assert.strict = Object.assign(strict, assert, {
```

Comment:
This will need a ponyfill to maintain support for the very old nodes and browsers that we target right now

Reply:
Is it ok to depend on [this module](https://github.com/sindresorhus/object-assign) and then `require` it?

I noticed other shims (Buffer/Object.keys) are dumped inline rather than added as dependencies. Is there a reason for that?